modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,58 @@
+
+* With this update, you can set the value of the `resourceTrackingMethod` to select the tracking method used by Argo CD. link:https://issues.redhat.com/browse/GITOPS-1862[GITOPS-1862]
+
+* With this update, you can add entries to the `argocd-cm` configmap using the `extraConfig` field of {gitops-title} Argo CD custom resource. The entries specified are reconciled to the live `config-cm` configmap without validations. link:https://issues.redhat.com/browse/GITOPS-1964[GITOPS-1964]
+
+* With this update, on {product-title} 4.11, the {gitops-title} **Environments Details** page in the {gitops-title} developer perspective shows history of the successful deployments of the application environments along with links to the revision for each deployment. link:https://issues.redhat.com/browse/GITOPS-1269[GITOPS-1269]
+
+* With this update, you can manage resources with Argo CD that are also being used as template resources or "source" by an Operator. link:https://issues.redhat.com/browse/GITOPS-982[GITOPS-982]
+
+* With this update, the Operator will now configure the Argo CD workloads with the correct permissions to satisfy the Pod Security Admission that has been enabled for Kubernetes 1.24. link:https://issues.redhat.com/browse/GITOPS-2026[GITOPS-2026]
+
+* With this update, Config Management Plugins 2.0 is supported. The Argo CD custom resource now allows to specify sidebar containers for the repo server. link:https://issues.redhat.com/browse/GITOPS-766[GITOPS-776]
+
+* With this update, all communication between the Argo CD components and Redis cache are properly secured using modern TLS encryption. link:https://issues.redhat.com/browse/GITOPS-720[GITOPS-720]
+
+* This release of {gitops-title} adds support for IBM Z and IBM Power on {product-title} 4.10.
+
+[id="fixed-issues-1-6-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, `system:serviceaccount:argocd:gitops-argocd-application-controller` cannot create resource "prometheusrules" in API group `monitoring.coreos.com` in the namespace `webapps-dev`. This update fixes this issue and {gitops-title} is now able to manage all resources from the `monitoring.coreos.com` API group. link:https://issues.redhat.com/browse/GITOPS-1638[GITOPS-1638]
+
+* Before this update, While reconciling cluster permissions, every time the secret was getting deleted if the secret belonged to a cluster config instance. This update fixes this issue and now the `namespaces` field from the secret is deleted instead of deleting the complete secret. link:https://issues.redhat.com/browse/GITOPS-1777[GITOPS-1777]
```

Comment:
```suggestion
* Before this update, when reconciling cluster permissions, the secret was always deleted if the secret belonged to a cluster config instance. This update fixes this issue and now the `namespaces` field from the secret is deleted instead of deleting the complete secret. link:https://issues.redhat.com/browse/GITOPS-1777[GITOPS-1777]
```